src/server.c

```diff
@@ -537,7 +537,10 @@ struct redisCommand redisCommandTable[] = {
 
     {"sintercard",sinterCardCommand,-3,
      "read-only @set",
-     0,sintercardGetKeys,0,0,0,0,0,0},
+     {{"read",
+       KSPEC_BS_INDEX,.bs.index={1},
+       KSPEC_FK_RANGE,.fk.range={0,1,1}}},
```

Comment:
that should actually be a `KSPEC_FK_KEYNUM` now. copy from ZINTERCARD rather than from SINTER

Reply:
right.. sorry my bad